setup.py

```diff
@@ -12,7 +12,9 @@
 test_requirements = [
     'pytest',
     'pytest-cov',
-    'mock'
+    'mock',
+    'numpy',
+    'scipy'
 ]
```

Comment:
The `test_requirements` have been dropped from the master branch in favor of a `dev-requirements.txt` file. As installing numpy / scipy is currently very costly under Linux (built from source when installed with pip or setuptools) I would rather not put them as a dependency in `dev-requirements.txt`) but instead upgrade the travis config to use miniconda, at least on some of the entries of the build matrix and only put those dependencies in the `.travis.yml` file.